http-client/http-client-core/src/commonMain/kotlin/ai/koog/http/client/KoogHttpClient.kt

```diff
@@ -27,6 +27,7 @@ public interface KoogHttpClient {
     public suspend fun <R : Any> get(
         path: String,
         responseType: KClass<R>,
+        parameters: Map<String, String>? = emptyMap(),
```

Comment:
If parameters are nullable, then the default value should be null. Otherwise:
```suggestion
        parameters: Map<String, String> = emptyMap(),
```

Let's make parameters non-nullable to simplify nternal logic